package/MDAnalysis/topology/guessers.py

```diff
@@ -36,6 +36,7 @@
 
 from ..lib import distances
 from . import tables
+from .tables import Z2SYMB
```

Comment:
As per my above comment, it would be simpler to just leave `guessers.py` as it was, I have a feeling that otherwise we'll unecessarily affect downstream user code.

Reply:
That being said the changes made here are probably quite useful, could you open a separate issue/PR set?

Reply:
My bad. I completely forgot to remove that line. 
Also, do you want me to do a changes in a new branch, commit it and then make a PR there?

Reply:
So I'm not 100% sure if there were any changes to `guessers.py` that was beyond just purely the scope of adding elements to the PDB parser. If so in the first instance I would open a new issue detailing what changes are being proposed and why. Then in a separate branch introduce those changes and open a PR.

Reply:
Sure, that makes sense. I also don't quite understand why TravisCI builds might be failing? Is there something I can do to improve on that?

Reply:
If you look at the Travis output: https://travis-ci.com/github/MDAnalysis/mdanalysis/jobs/303605080, it looks like it's failing because of the changes that were made to `guessers.py`. Since we'll probably be reverting those changes here, these tests should stop failing. However, it would be worth noting these failures if you plan to introducing any changes to `guessers.py` in the future.